src/containerapp/azext_containerapp/containerapp_java_decorator.py

```diff
@@ -0,0 +1,133 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from typing import Any, Dict
+
+from azure.cli.command_modules.containerapp._utils import safe_get, safe_set
+from azure.cli.core.commands import AzCliCommand
+from azure.cli.core.azclierror import ValidationError, ResourceNotFoundError
+from azure.cli.command_modules.containerapp.base_resource import BaseResource
+from knack.log import get_logger
+
+from azure.cli.command_modules.containerapp._utils import _get_existing_secrets
+from knack.util import CLIError
+
+from ._models import JavaLoggerSetting
+
+from ._client_factory import handle_raw_exception, handle_non_404_status_code_exception
+
+logger = get_logger(__name__)
+
+
+class ContainerappJavaLoggerDecorator(BaseResource):
+
+    def __init__(self, cmd: AzCliCommand, client: Any, raw_parameters: Dict, models: str):
+        super().__init__(cmd, client, raw_parameters, models)
+        self.containerapp_def = None
+        try:
+            self.containerapp_def = self.client.show(cmd=cmd,
+                                                resource_group_name=raw_parameters.get("resource_group_name"),
+                                                name=raw_parameters.get("name"))
```

Comment:
PEP 8: E128 continuation line under-indented for visual indent
![image](https://github.com/Azure/azure-cli-extensions/assets/46143499/23c32bdd-801b-4133-a504-28a740c45b28)